mne/io/utils.py

```diff
@@ -84,7 +84,7 @@ def _mult_cal_one(data_view, one, idx, cals, mult):
         one.shape[1],
     )  # noqa: E501
     if mult is not None:
-        mult.ndim == one.ndim == 2
```

Comment:
I think this should have the assert, seems like a mistake

Reply:
feel free to change to `assert` if you're feeling brave. Otherwise please leave this cruft in, add a `# TODO: change to assert`  and I will try it in a follow-up PR

Reply:
Pretty sure it should be that way. Also else the line doesn't do anything than to produce a bool that get's immediatley discarded.